app/scripts/controllers/overview.js

```diff
@@ -565,6 +565,33 @@ angular.module('openshiftConsole')
       MetricsService.isAvailable(true).then(function(available) {
         $scope.showMetrics = available;
       });
+
+      $scope.$on('metrics-connection-failed', function(e, data) {
+        var hidden = AlertMessageService.isAlertPermanentlyHidden('metrics-connection-failed');
+        if (hidden || $scope.alerts['metrics-connection-failed']) {
+          return;
+        }
+
+        $scope.alerts['metrics-connection-failed'] = {
+          type: 'warning',
+          message: 'An error occurred getting metrics.',
+          links: [{
+            href: data.url,
```

Comment:
Is this opening the particular request that failed, or is it going to the splash page?